setup.cfg

```diff
@@ -1,5 +1,42 @@
 [metadata]
+name=url
+version=0.4.2a
+description = URL parsing, sanitization, and other handy features.
+long_description = 
+    Some helper functions for parsing URLs, sanitizing them, normalizing   them.
+    
+    This includes support for escaping, unescaping, punycoding,   unpunycoding, cleaning parameter and query strings, and a little more sanitization.
 description-file = README.md
+author = Dan Lecoq
+author_email = dan@moz.com
```

Comment:
At this point, we should probably change that. @lindseyreno @tammybailey - is there a better email? Maybe a team email or a `opensource@moz.com` would be appropriate?

Reply:
`turbo@moz.com` is probably the best right now

Reply:
I'd suggest shifting to a 'corporate' address/name since PyPI shows @dlecocq and @b4hand as the owners at the moment.

Reply:
```suggestion
author_email = turbo@moz.com
```

Reply:
👍